File: Heat/ViewModels/Invoices/EditInvoiceViewModel.vb
﻿Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Invoices
    Public Class EditInvoiceViewModel
        <Key> _
        Property ID As Integer

        <Display(name:="Cliente")> _
        Property CustomerName As String

        <Display(name:="Numero documento")> _
        Property InvoiceNumber As String

        <Display(name:="Data documento")> _
        Property InvoiceDate As String

        Property Rows As List(Of PresentationInvoiceRowViewModel)


    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/Invoices/EditInvoiceViewModel.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Invoices
{
    public class EditInvoiceViewModel
	{
		[Key()]
		public int ID { get; set; }

		[Display(Name = "Cliente")]
		public string CustomerName { get; set; }

		[Display(Name = "Numero documento")]
		public string InvoiceNumber { get; set; }

		[Display(Name = "Data documento")]
		public string InvoiceDate { get; set; }

		public List<PresentationInvoiceRowViewModel> Rows { get; set; }


	}
}

